src/component/InvoiceContainer/index.jsx

```diff
@@ -0,0 +1,33 @@
+import { useEffect, useState } from "react"
+import Invoice from "../Invoice"
+import Footer from "../Footer"
+import Header from "../Header"
+
+
+function InvoiceContainer() {
+    const [invoices, setInvoices] = useState([])
+    useEffect(() => {
+        // Instructions here on what you need to happen on component mount
+        fetch('https://invoicing-api.dev.io-academy.uk/invoices')
+            .then(res => res.json())
+            .then((invoiceData) => {
+                setInvoices(invoiceData.data)
+
+            })
+    }, [])
+
+    return (
+        <>
+            <Header />
```

Comment:
The header should display on every page, so you should display it in app outside the routes instead